src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let totalCost;
+
+  if (days < 3) {
+    totalCost = dayCost * days;
+
+    return totalCost;
+  }
+
+  if (days >= 3 && days <= 6) {
+    totalCost = dayCost * days - 20;
+
+    return totalCost;
+  }
+
+  if (days >= 7) {
+    totalCost = (dayCost * days) - 50;
```

Comment:
It would be better to create a variable with a value days * 40, because you use it several times